hkm/static/hkm/js/hkm-v2.js

```diff
@@ -337,7 +343,50 @@ palikka
       });
     }
   }
+})
+.define('app.reset-password', ['jQuery', 'docReady'], function() {
+  $('#password-reset-email').on('submit', function(event) {
+    event.preventDefault();
 
+    $.post('/', {
+      action: 'password_reset',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      email: $(this).find('input[name="email"]').val(),
```

Comment:
```suggestion
      email: event.target.elements.email.value,
```
Here are some examples about how you can access the value of the field by using the on submit event:

https://jsfiddle.net/v5o1ptdz/

I'm not sure if there are some jQuery or browser specific quirks to take into account.